bundles/framework/announcements/instance.js

```diff
@@ -27,15 +29,14 @@ Oskari.clazz.define('Oskari.framework.bundle.announcements.AnnouncementsBundleIn
             if (me.started) {
                 return;
             }
-
+            this.tile = me.plugins['Oskari.userinterface.Tile'] ? me.plugins['Oskari.userinterface.Tile'] : null;
```

Comment:
DefaultExtension has `getTile()` that could be used here. We shouldn't store the reference here but use the getter instead.